uint32 id
float64 start
float64 length
